projectName-selenium/src/test/resources/META-INF/spring/selenium.properties

```diff
@@ -1,3 +1,3 @@
 selenium.serverUrl=http://localhost:8080
 selenium.contextName=projectName-web
-selenium.applicationContextUrl=${selenium.serverUrl}/${selenium.contextName}
+selenium.applicationContextUrl=${selenium.serverUrl}/${selenium.contextName}
```

Comment:
- [ ] Please restore this file to its original state.

Reply:
returned original state.
5c49c63